store/tikv/2pc.go

```diff
@@ -1057,12 +1047,22 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) error {
 				c.cleanWg.Done()
 			}()
 		}
+		c.txn.commitTS = c.commitTS
+		if binlogSkipped {
+			binloginfo.RemoveOneSkippedCommitter()
+		} else {
+			if err != nil {
+				c.writeFinishBinlog(ctx, binlog.BinlogType_Rollback, 0)
+			} else {
+				c.writeFinishBinlog(ctx, binlog.BinlogType_Commit, int64(c.commitTS))
+			}
+		}
 	}()
 
 	binlogChan := c.prewriteBinlog(ctx)
 	prewriteBo := NewBackoffer(ctx, PrewriteMaxBackoff).WithVars(c.txn.vars)
```

Comment:
if panic happened after `prewriteBinlog` before `<-binlogChan` will miss `RemoveOneSkippedCommitter`

Reply:
Yes, can not avoid it, I've added a reset operation.

Reply:
if there is in-flight txn, calling reset will make SkippedCommitterCounter < 0 finally?

Reply:
Yes, `reset` is used for troubleshooting, if sth wrong with it, e.g. counter != 0 forever after resetting, we could reset it once again. I think we could not avoid it to happen, but hopefully, it will never be used.